hang_man.py

```diff
@@ -0,0 +1,148 @@
+import os
+import sys
+import copy
+import string
+import random
+import argparse
+import urllib.request
+
+
+class Hangman:
+
+    def __init__(self, min_limit):  # Creating init method that takes in max word limit as perimeter
+        self.min_limit = min_limit  # Minimum word limit taken from user
+        self.word_list = []  # A list that stores all the word from local file or web
+        self.right_list = []  # A list that stores the right word
+        self.guess_letter = str  # User input of their guessed character
+
+    def access_doc(self):
+        file_loc = './word_data/words.txt'
+        file_dir = './word_data/'
+        url = 'http://norvig.com/ngrams/sowpods.txt'  # Create a variable that stores for URL for text file
+        file_exist = os.path.exists(file_loc)  # Boolean value to check if text file exist first
```

Comment:
fyi there are functions such as os.path.dirname to get the directory of a file. so you could have defined the file path and then to check if the folder existed use the dirname fxn. would have been a bit dryer. you can see for a small project like this it doesn;t make a huge difference, but for a huge project with thousands or millions of loc (lines of code) this can really add up